index.html

```diff
@@ -861,19 +861,26 @@ <h2 id='ldpcvpost'>HTTP POST</h2>
       <section id="solid-ldp-acls">
         <h2>ACLs are LDP RDF Sources</h2>
         <p>
-          The <a>ACL</a> for a controlled resource on a conforming server MUST itself be an LDPRS. This <a>ACL</a>
-          resource SHOULD be located in the same server as the controlled resource.
+          An <a>ACL</a> for a controlled resource on a conforming server MUST itself be an <a>LDP-RS</a>.
         </p>
       </section>
       <section id="link-rel-acl">
         <h2>ACLs are discoverable via Link Headers</h2>
         <p>
-          A conforming server MUST advertise the appropriate <a>ACL</a> for every controlled resource in HTTP
-          responses with a <code>Link: rel="acl"</code> header. This interaction pattern is described in
-          [[!SOLIDWEBAC]]
+          The interaction pattern is for ACL discovery is described in [[!SOLIDWEBAC]]
```

Comment:
Grammatical typo?
`The interaction pattern [is] for ACL discovery is described...`

Reply:
Indeed